app/code/core/Mage/Core/Model/Session/Abstract/Varien.php

```diff
@@ -89,68 +98,43 @@ public function start($sessionName = null)
         }
 
         $cookie = $this->getCookie();
-        if (Mage::app()->getStore()->isAdmin()) {
-            $sessionMaxLifetime = Mage_Core_Model_Resource_Session::SEESION_MAX_COOKIE_LIFETIME;
-            $adminSessionLifetime = Mage::getStoreConfigAsInt('admin/security/session_cookie_lifetime');
-            if ($adminSessionLifetime > $sessionMaxLifetime) {
-                $adminSessionLifetime = $sessionMaxLifetime;
-            }
-            if ($adminSessionLifetime > 60) {
-                $cookie->setLifetime($adminSessionLifetime);
-            }
-        }
 
-        // session cookie params
-        $cookieParams = [
-            'lifetime' => (int)$cookie->getLifetime(),
-            'path'     => $cookie->getPath(),
-            'domain'   => $cookie->getConfigDomain(),
-            'secure'   => $cookie->isSecure(),
-            'httponly' => $cookie->getHttponly()
-        ];
-
-        if (!$cookieParams['httponly']) {
-            unset($cookieParams['httponly']);
-            if (!$cookieParams['secure']) {
-                unset($cookieParams['secure']);
-                if (!$cookieParams['domain']) {
-                    unset($cookieParams['domain']);
-                }
+        // Migrate old cookie from 'frontend'
+        if ($sessionName === Mage_Core_Controller_Front_Action::SESSION_NAMESPACE) {
+            if ($cookie->get('frontend') && !$cookie->get($sessionName)) {
+                $_COOKIE[$sessionName] = $cookie->get('frontend');
+                $cookie->delete('frontend');
```

Comment:
@justinbeaty do we need this code? I mean, is the "frontend" cookie still relevant?

Reply:
If someone upgraded from Magento 1.9 -> Maho, it is still needed. But not if they went Magento 1.9 -> OpenMage -> Maho.

I would have liked to put that code in an Observer anyway. Especially to support if the user wants to change the cookie name themselves. But was just an unnecessary change in this PR.

Reply:
as first sight I'd say "duck it", I mean, if somebody still has magento1 I think session cookies are the last of their problem and I'd nuke them

Reply:
I had to re-read to see why the change was introduced in the first place. https://github.com/OpenMage/magento-lts/issues/990

So, we could also consider to use the same code to rename to “maho_frontend”, or maybe something different to make it so we aren’t recognized as OM.

Reply:
I didn't recognize if was for making it recognizable! then sure I'd use something like maho_frontend, maho_fe or something like that

Reply:
How about just “maho_session”? I think it’s probably more descriptive to a customer than “maho_frontend”, especially since these days many sites have a cookie policy page that lists the names of all cookies.

The admin cookie would still just be “adminhtml”.